iquip/apps/explorer.py

```diff
@@ -31,6 +30,45 @@ def __init__(self, parent: Optional[QWidget] = None):
         layout.addWidget(self.openButton)
 
 
+class FileFinderThread(QThread):
+    """QThread for finding the file list using a command line.
+
+    Signals:
+        finished(experimentList, parent): Fetching the file list is finished.
+
+    Attributes:
+        path: The path of the directory to search experiment files.
+        parent: The widget corresponding to the path.
+    """
+
+    finished = pyqtSignal(list, object)
```

Comment:
There is already a signal with the same name: [`QThread.finished()`](https://doc.qt.io/qt-5/qthread.html#finished)

Reply:
Yes! Actually, I intended it.

Probably, the `finished` signal was created with the aim of announcing the end of the thread.
I needed the similar signal which has additional arguments, so I named it the same.

But, if you want to remain the original signal, I will rename it lik `fetched`.

Reply:
Hmm, however, `finished` signal is emitted when a `QThread` finishes executing, after stopping the event loop.
I am not sure if it is safe to overwrite such a private signal which already has its own role (since Python does not support overloading).
If we don't have to name it `finished`, I would prefer to use another name!

Reply:
Okay! Then, I will change the name into another one.